test/src/test/java/jenkins/model/JenkinsTest.java

```diff
@@ -705,4 +708,63 @@ public void versionIsSavedInSave() throws Exception {
     public void jobCreatedByInitializerIsRetained() {
         assertNotNull("JENKINS-47406 should exist", j.jenkins.getItem("JENKINS-47406"));
     }
+
+    @Issue("SECURITY-2047")
+    @Test
+    public void testLogin123() throws Exception {
+        j.jenkins.setSecurityRealm(j.createDummySecurityRealm());
+        j.jenkins.setAuthorizationStrategy(new MockAuthorizationStrategy());
+        WebClient wc = j.createWebClient();
+
+        try {
+            wc.goTo("login123");
+            fail("Page should be protected.");
+        } catch (FailingHttpStatusCodeException e) {
+            assertThat(e.getStatusCode(), is(403));
+        }
+    }
+
+    @Issue("SECURITY-2047")
+    @Test
+    public void testLogin123WithRead() throws Exception {
```

Comment:
this test is failing on CI
https://github.com/jenkinsci/jenkins/pull/5288/checks?check_run_id=1904565850

Reply:
Thanks Tim for the notice. Some tests were failing locally but I hoped it was just due to my local setup (and continuous modifications). Will adjust it soon-ish 👍